srcpkgs/apparmor/files/profiles/usr.bin.dhcpcd

```diff
@@ -58,5 +58,5 @@ profile dhcpcd /{usr/,}bin/dhcpcd {
   owner @{PROC}/@{pid}/stat r,
 
   # Site-specific additions and overrides. See local/README for details.
-  #include <local/usr.bin.dhcpcd>
+  include if exists <local/usr.bin.dhcpcd>
```

Comment:
is this necessary or just recommended?

Reply:
thanks for the information... just needed to know for some of the packages I maintain.